src/test/java/Homework17.java

```diff
@@ -0,0 +1,46 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class Homework17 extends BaseTest {
+    @Test
+    public void addSongToPlaylist() throws InterruptedException   {
+        openUrl();
+        enterEmail("demo@class.com");
+        enterPassword("te$t$tudent");
+        clickLoginBtn();
+        enterToSearch("Epic song");
+        //click View All button results
+        WebElement viewAllBtn = driver.findElement(By.cssSelector("[data-test='view-all-songs-btn']"));
+        viewAllBtn.click();
+        //click first song from list
+        List<WebElement> songsInResults = driver.findElements(By.cssSelector(".search-results .items"));
+        songsInResults.get(0).click();
+        // click Add To button
+        WebElement addToBtn = driver.findElement(By.cssSelector(".btn-add-to"));
+        addToBtn.click();
+        //Creation of new playlist with selected song
+        WebElement playlistNameFld = driver.findElement(By.cssSelector("#songResultsWrapper [required]"));
+        playlistNameFld.click();
+        playlistNameFld.sendKeys("Great Music_777");
+        WebElement savePlaylistBtn = driver.findElement(By.cssSelector("#songResultsWrapper [type='submit']"));
+        savePlaylistBtn.click();
+        //assertion - success banner
+        WebElement successBanner = driver.findElement(By.cssSelector(".success"));
+        Assert.assertTrue(successBanner.isDisplayed());
+    }
+       public void enterToSearch (String song) throws InterruptedException {
+           WebElement searchLine = driver.findElement(By.cssSelector("[name='q']"));
+           searchLine.click();
+           searchLine.clear();
+           searchLine.sendKeys(song);
+       }
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._